app/init.php

```diff
@@ -90,9 +90,10 @@
 Config::load('storage-inputs', __DIR__.'/config/storage/inputs.php'); 
 Config::load('storage-outputs', __DIR__.'/config/storage/outputs.php'); 
 
-$auth = App::getEnv('_APP_REDIS_AUTH',null);
-if($auth != null) {
-    Resque::setBackend('redis://'.App::getEnv('_APP_REDIS_AUTH', '').'@'.App::getEnv('_APP_REDIS_HOST', '').':'.App::getEnv('_APP_REDIS_PORT', ''));
+$user = App::getEnv('_APP_REDIS_USER','');
+$pass = App::getEnv('_APP_REDIS_PASS','');
+if(!empty($pass)) {
```

Comment:
We should allow authentication if user *or* password have been set.

Reply:
Updated